packages/legend-extension-dsl-persistence/src/graphManager/protocol/pure/DSL_Persistence_PureProtocolProcessorPlugin.ts

```diff
@@ -401,4 +528,130 @@ export class DSL_Persistence_PureProtocolProcessorPlugin
       },
     ];
   }
+
+  V1_getExtraAssertionStatusBuilders(): V1_AssertionStatusBuilder[] {
+    return [
+      (
+        element: V1_AssertionStatus,
+        atomicTest: AtomicTest,
+        plugins: PureProtocolProcessorPlugin[],
+      ): AssertionStatus | undefined => {
+        if (element instanceof V1_AllRowsEquivalentToJsonAssertFail) {
+          let assertion = atomicTest.assertions.find(
+            (a) => a.id === element.id,
+          );
+          const extraAssertionBuilder = plugins.flatMap(
+            (plugin) =>
+              (
+                plugin as Testable_PureProtocolProcessorPlugin_Extension
```

Comment:
why do you call these plugins here? Is it to build the metamodel for `AllRowsEquivalentToJsonAssertion`?

Reply:
Here it is building extra testable assertions, not assertion status. The assertion that is retrieved and built from the plugins is then used in the building of the assertion status (line 557 - 563).   
I got this pattern from all the other assertion statuses - seems like this extra step is needed for retrieving the correct assertion to be used. Please refer to `buildEqualToJsonAssertFail` (or any other builder for assertion statuses) in V1_TestResultBuilderHelper.ts`